apps/weshipit/pages/consulting.tsx

```diff
@@ -18,14 +25,16 @@ export function Consulting(props: ConsultingProps) {
     <Layout
       withContainer
       withFooter
-      seoTitle="React Native Consulting"
-      seoDescription="Looking to improve your skills as a React Native developer? Our coaching services are here to help. Our experienced team will provide personalized guidance and support to help you take your skills to the next level. Whether you're a beginner or an experienced developer, we have a coaching program that's right for you."
       ogImageTitle="Helping you to ship your vision in production"
       withAccessoryRight={
         <Button intent="ghost" href="https://cal.com/davidl/coaching">
           Schedule 1-to-1 consulting
         </Button>
       }
+      seoTitle={'React Native Consulting'}
```

Comment:
instead of dupplicating the content you can use metadata.title and metadata.description